kotlin-react-dom-legacy/src/main/kotlin/react/dom/createPortal.ext.kt

```diff
@@ -10,6 +11,6 @@ fun createPortal(
     handler: Render,
 ): ReactPortal =
     createPortal(
-        children = createElement(handler)!!,
+        children = createElement<Props>(handler)!!,
```

Comment:
double exclamation mark is not needed here because `createPortal` should receive nullable `ReactNode`

Reply:
Separate PR?

Reply:
You are right - I will fix it in separate PR